src/validations/test/styleguides/sch.xspec

```diff
@@ -62,14 +62,22 @@
                     label="that is correct" />
             </x:scenario>
             <x:scenario
-                label="when the id attribute is absent">
+                label="when the id attribute is absent from a report">
+                <x:context>
+                    <sch:report
+                        xmlns="http://purl.oclc.org/dsdl/schematron"
+                        label="test label">Missing ID.</sch:report>
+                </x:context>
+                <x:expect-assert
+                    id="has-id-attribute"
+                    label="that is an error" />
+            </x:scenario>
+            <x:scenario
+                label="when the id attribute is absent from an assert">
                 <x:context>
                     <sch:assert
-                        diagnostics="has-id-attribute-diagnostic"
-                        role="error"
-                        sqf:fix="add-id"
-                        test="@id"
-                        xmlns="http://purl.oclc.org/dsdl/schematron">Has id.</sch:assert>
+                        xmlns="http://purl.oclc.org/dsdl/schematron"
+                        label="test label">Missing ID.</sch:assert>
```

Comment:
The (below) `expect-assert` on this `sch:assert` does not fire. @GaryGapinski thoughts why?

The rule _should_ be matching on both `sch:assert` and `sch:report`. See here: https://github.com/18F/fedramp-automation/blob/has-id-attribute-debugging/src/validations/styleguides/sch.sch#L89-L90